README.md

```diff
@@ -1 +1,236 @@
-# spa-booking-backend
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>The Spa Api</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 The Spa Api ](#-the-spa-api-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+    - [ Video Presentation](#video)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Get into development](#get-into-development)
+    - [Run test](#run-test)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 The Spa Api <a name="about-project"></a>
+
+**The Spa Api** is the backend rails Api for the spa app built to help individuals easily book their spa sessions.
+
+## 🛠 Link to back-end <a name="link-frontend"></a>
+
+-- [Link to frontend-end!](https://github.com/newhoteng/spa-booking-frontend)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgres.com">Postgres</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Set up controller**
+- **Authentication with devise**
+<!-- - **Authorization with cancan** -->
+- **View the all services available**
+- **Users can add services**
+- **Users can make reservation of services**
+- **Users can view all reservations they have made**
+- **Users can delete services**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo]()
+
+<!-- Presentation -->
+
+## Video Presentation <a name="video"></a>
+
+-[Presentation Video]()
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Visual Studio Code.
+- Ruby
+- Git bash.
+- GitHub Account.
+
+### Setup
+
+Install Ruby
+Go to github and find the repository The Spa Api
+Click on code and copy then go to your gitbash cli on your computer Clone this repository to your desired folder
+
+```sh
+  cd my-folder
+  git clone https://github.com/newhoteng/spa-booking-backend
+  bundle install
+
+```
+
+### Install
+
+```
+git clone https://github.com/newhoteng/spa-booking-backend
+```
+
+### Usage
```

Comment:
You should probably mention the tests somewhere here. Als oconsider mentioning the documentation (maybe not here, but somehwere) :)